src/components/Categories.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { checkStatus } from '../redux/categories/categories';
+
+export default function Categories() {
+  const dispatch = useDispatch();
+  const status = useSelector((state) => state.categories);
+  const handleCheckStatus = (event) => {
+    dispatch(checkStatus(event.target.getAttribute('id')));
+  };
```

Comment:
- [ ] The `checkStatus` method does not need any argument passed to it. Passing `event.target.getAttribute('id')` to it now become unneeded. Please call it without passing an argument. Something like this 👇🏼 should be alright:

  ```JavaScript
  const handleCheckStatus = () => {
    dispatch(checkStatus());
  };
  ```